data/service/api/v1/alerts.go

```diff
@@ -20,8 +19,8 @@ import (
 func AlertsRoutes() []service.Route {
 	return []service.Route{
 		service.MakeRoute("GET", "/v1/alerts/:userID/:followedID", Authenticate(GetAlert)),
-		service.MakeRoute("POST", "/v1/alerts/:userID", Authenticate(UpsertAlert)),
-		service.MakeRoute("DELETE", "/v1/alerts/:userID", Authenticate(DeleteAlert)),
+		service.MakeRoute("POST", "/v1/alerts/:userID/:followedID", Authenticate(UpsertAlert)),
```

Comment:
It is, yes. It is the user id of person being followed. Or to put it another way, it is the user id of the person sharing their data.

I had originally named it "invitorID", but after some discussion with Todd, we changed it to "followedID". You can peruse that [history here](https://github.com/tidepool-org/platform/pull/657#discussion_r1331482131).

Reply:
Consider `followedUserID` to make it clear it is a user id that is being followed.

Reply:
Yea, that would have been clearer if implemented at the beginning, but we didn't think about that. At this point it would be a very large change to a large chunk of code, so probably something we should avoid.